src/ex1/style.css

```diff
@@ -0,0 +1,55 @@
+body{
+    background-color: aliceblue;
+}
+
+.todo{
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    box-shadow: 0 2px 2px rgba(0, 0, 0, 0.05);
+    padding: 20px;
+    margin-bottom: 20px;
+    text-align: center;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+}
+
+.todoList{
+    justify-content: center;
```

Comment:
there's no effect to justify-content/align-items/flex-direction if the element is not with display:flex/grid